File: MODUL1/PraktikumPy/PRAK104-2310817110004-Abdurrahman Gilang Harjuna.py
hargaA = 400000
hargaB = 350000
diskonA = 0.13
diskonB = 0.21
hargaDiskonA = hargaA * (1 - diskonA)
hargaDiskonB = hargaB * (1 - diskonB)
print(f"Harga sepatu A adalah {hargaA}")
print(f"Harga sepatu B adalah {hargaB}")
print(f"Sepatu A mendapat diskon {diskonA*100:.0f}% sehingga harganya menjadi {hargaDiskonA:.0f}")
print(f"Sepatu B mendapat diskon {diskonB*100:.0f}% sehingga harganya menjadi {hargaDiskonB:.0f}")

File: MODUL1/PraktikumC/PRAK104-2310817110004-AbdurrahmanGilangHarjuna.c
#include <stdio.h>

int main() {
    int hargaA = 400000;
    int hargaB = 350000;    
    float diskonA = 0.13;
    float diskonB = 0.21;
    int hargaDiskonA = hargaA * (1 - diskonA);
    int hargaDiskonB = hargaB * (1 - diskonB);    
    printf("Harga sepatu A adalah %d\n", hargaA);
    printf("Harga sepatu B adalah %d\n", hargaB);
    printf("Sepatu A mendapat diskon %.0f%% sehingga harganya menjadi %d\n", diskonA * 100, hargaDiskonA);
    printf("Sepatu B mendapat diskon %.0f%% sehingga harganya menjadi %d\n", diskonB * 100, hargaDiskonB);    
    return 0;
}
